setup.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python3
 
 from setuptools import setup
+import sys
+
+# Requires python >= 3.3
+assert sys.version_info.major >= 3
+assert sys.version_info.minor >= 3
```

Comment:
Yes, I agree that blocking installation on incompatible pythons is a good idea. I'd go a step further and inform the user of _why_ it crashes so they can quickly work around it.

I meant to say "there's still no clear error to the end-user".